homeassistant/components/livebox/__init__.py

```diff
@@ -0,0 +1,149 @@
+"""Orange Livebox."""
+import logging
+from datetime import timedelta
+import voluptuous as vol
+
+from aiosysbus import Sysbus
+from aiosysbus.exceptions import HttpRequestError, AuthorizationError
+
+from homeassistant.const import CONF_HOST, CONF_PORT, CONF_USERNAME, CONF_PASSWORD
+from homeassistant.helpers import config_validation as cv, device_registry as dr
+
+from .const import DOMAIN, DEFAULT_USERNAME, DEFAULT_HOST, DEFAULT_PORT
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.Schema(
+            {
+                # Validate as IP address and then convert back to a string.
+                vol.Required(CONF_HOST, default=DEFAULT_HOST): cv.string,
+                vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+                vol.Optional(CONF_USERNAME, default=DEFAULT_USERNAME): cv.string,
+                vol.Required(CONF_PASSWORD): cv.string,
+            }
+        )
+    },
+    extra=vol.ALLOW_EXTRA,
+)
+
+SCAN_INTERVAL = timedelta(minutes=1)
+
+
+async def async_setup(hass, config):
+    """Load configuration for Livebox component."""
+
+    return True
```

Comment:
Please add an import step to the config flow and start the config flow import step from here.